test/unittest/run_all.cmd

```diff
@@ -0,0 +1,38 @@
+@echo off
+setlocal ENABLEDELAYEDEXPANSION
+
+@REM This script is intended to be used either by the CI or by a developer that
+@REM wants to check that no unit tests have been broken to ensure there are no
+@REM regressions. At the end, it reports how many tests were successful.
+
+set SCRIPT_DIR=%~dp0
+set GRAVITY_BIN=%SCRIPT_DIR%/../../gravity_visualstudio/bin/cli/x64/Debug/gravity.exe
+set /a tests_total=0
+set /a tests_success=0
+set /a tests_fail=0
+set /a tests_timeout=0
```

Comment:
I am unsure what the best replacement is for timeout, so I removed the timout checks for now.